### ISSUE TITLE
Change exchange dependency in DataFeed

### ISSUE THREAD

**Issue by ramilexe** [rodrigo-brito/ninjabot#257]
`DataFeedSubscription` accepts `exchange service.Exchange` but uses only `CandlesSubscription` method from `Feeder` interface. Does it make sense to change dependency from Exchange interface to `Feeder`?

**Comment by rodrigo-brito** [comment]
Of course, both will be compatible. But in theory they are internal modules, this should not affect external users, are you having any problems with this point?

**Comment by ramilexe** [comment]
yes, I would like to use `future.WsCombinedKlineServe` method and instead of just implementing `CandlesSubscription` method and some other from `Feeder` I also need to implement `Broker` methods as well.

**Comment by rodrigo-brito** [comment]
Please, can you open a PR with this change?

**Comment by ramilexe** [comment]
@rodrigo-brito here we go https://github.com/rodrigo-brito/ninjabot/pull/258
